FrejaEidClient/src/main/java/com/verisec/frejaeid/client/enums/UserCustodianshipStatusResult.java

```diff
@@ -0,0 +1,37 @@
+package com.verisec.frejaeid.client.enums;
+
+/**
+ *
+ * @author veiggnu
+ */
+public enum UserCustodianshipStatusResult {
+    /**
+     * Custodianship status when the user has a custodian.
+     */
+    USER_HAS_CUSTODIAN("USER_HAS_CUSTODIAN"),
+    /**
+     * Custodianship status when we do not have a registered custodian
```

Comment:
Custodianship status when it's unknown if the user has a custodian.

Reply:
Done